concepts/05 UI Components/DataGrid/20 Editing/43 Customize Edit Form/Customize Edit Form.md

```diff
@@ -0,0 +1 @@
+DataGrid uses the [Form](/Documentation/Guide/UI_Components/Form/Getting_Started_with_Form/) component for editing in the [form](/Documentation/Guide/UI_Components/DataGrid/Editing/#User_Interaction/Form_Mode) and [popup](/Documentation/Guide/UI_Components/DataGrid/Editing/#User_Interaction/Popup_Mode) edit modes. The component automatically configures the built-in form based on the column and editing settings. In general, you can use the [Form API](/Documentation/ApiReference/UI_Components/dxForm/) to work with the built-in forms. However, you need to take into account the DataGrid specificities.
```

Comment:
When a named mode is used as an aspect of circumstances - in X mode, you need no article.

```suggestion
DataGrid uses the [Form](/Documentation/Guide/UI_Components/Form/Getting_Started_with_Form/) component for editing in [form](/Documentation/Guide/UI_Components/DataGrid/Editing/#User_Interaction/Form_Mode) and [popup](/Documentation/Guide/UI_Components/DataGrid/Editing/#User_Interaction/Popup_Mode) edit modes. The component automatically configures the built-in form based on column and editing settings. In general, you can use the [Form API](/Documentation/ApiReference/UI_Components/dxForm/) to work with built-in forms. However, you need to take DataGrid specifics into account.
```